tools/crm_resource_runtime.c

```diff
@@ -1784,19 +1784,6 @@ cli_resource_execute_from_params(pcmk__output_t *out, const char *rsc_name,
      */
     setenv("OCF_TRACE_FILE", "/dev/stderr", 0);
 
-    if (override_hash) {
-        GHashTableIter iter;
-        char *name = NULL;
-        char *value = NULL;
-
-        g_hash_table_iter_init(&iter, override_hash);
-        while (g_hash_table_iter_next(&iter, (gpointer *) & name, (gpointer *) & value)) {
-            out->info(out, "Overriding the cluster configuration for '%s' with '%s' = '%s'",
-                      rsc_name, name, value);
-            g_hash_table_replace(op->params, strdup(name), strdup(value));
-        }
-    }
```

Comment:
We just want to drop the message, we still need the g_hash_table_replace() which does the actual overriding ...